src/plugins/discover/public/index.ts

```diff
@@ -16,4 +16,5 @@ export function plugin(initializerContext: PluginInitializerContext) {
 
 export { SavedSearch, SavedSearchLoader, createSavedSearchesLoader } from './saved_searches';
 export { ISearchEmbeddable, SEARCH_EMBEDDABLE_TYPE, SearchInput } from './application/embeddable';
+export { getSharingData } from './application/helpers/get_sharing_data';
```

Comment:
why do we need to export this ?

Reply:
didn't test but since this is used in the x-pack code, it might be the reason for the larger bundle size? since it's now exported on top level of the `public` folder?

Reply:
I changed this to do an async import from the reporting code that needs this helper function